docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -0,0 +1,129 @@
+[id="configuring-a-ceph-backend_{context}"]
+
+= Configuring a Ceph backend
+
+If the original deployment uses a Ceph storage backend for any service
+(e.g. Glance, Cinder, Nova, Manila), the same backend must be used in the
+adopted deployment and custom resources (CRs) must be configured accordingly.
+
+If you use {CephService_first_ref}, on TripleO environments, the CephFS driver in {CephService} is configured to use
+its own keypair. For convenience, modify the `openstack` user so that you
+can use it across all {osp_prev_long} services.
+
+Using the same user across the services serves two purposes:
+
+* The capabilities of the user required to interact with {CephService}
+became far simpler and hence, more became more secure with RHOSP 18.
```

Comment:
do we need a variable for `RHOSP 18` ?

Reply:
This should probably be Red Hat OpenStack Services on OpenShift (RHOSO) 18.0. We no longer use RHOSP in 18.0.

Reply:
Fixed.